core/cautils/rbac.go

```diff
@@ -48,32 +48,40 @@ func (rbacObjects *RBACObjects) ListAllResources() (map[string]workloadinterface
 	return allresources, nil
 }
 
+func mapToMetadata(m map[string]interface{}) workloadinterface.IMetadata {
+	return workloadinterface.NewBaseObject(m)
+}
+
```

Comment:
question: did you mean to use this function in the SA2Wlid mapping?

I looked for where this function is referenced and failed to find anything: gopls lists no references to this function and staticcheck complains about it not being used.

Perhaps, you meant to use the function in the included refactoring of the WLID mapping?

Reply:
You are right, I forgot to remove the function